plugin/trino-databend/src/main/java/io/trino/plugin/databend/DatabendConfig.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.databend;
+
+import io.airlift.configuration.Config;
+import io.trino.plugin.jdbc.BaseJdbcConfig;
+
+import java.time.Duration;
+
+public class DatabendConfig
+        extends BaseJdbcConfig
+{
+    private Duration connectionTimeout = Duration.ofSeconds(60);
+
+    public Duration getConnectionTimeout()
+    {
+        return connectionTimeout;
+    }
+
+    @Config("databend.connection_timeout")
```

Comment:
We use `-` for config properties. 
```suggestion
    @Config("databend.connection-timeout")
```